apps/aeutils/test/aeu_env_tests.erl

```diff
@@ -28,7 +28,15 @@ extra_checks_test_() ->
      [{"User configuration cannot contain both 'mining > cuckoo > edge_bits' and deprecated 'mining > cuckoo > miner'",
        fun deprecated_miner_section_conflicting_with_edge_bits/0},
       {"User configuration cannot contain both 'mining > cuckoo > miners' and deprecated 'mining > cuckoo > miner'",
-       fun deprecated_miner_section_conflicting_with_miners/0}]
+       fun deprecated_miner_section_conflicting_with_miners/0},
+      {"User configuration cannot contain 'fork_management > fork > signalling_start_height' greater or equal to 'fork_management > fork > signalling_end_height'",
+       fun invalid_fork_signalling_interval/0},
+      {"User configuration cannot contain 'fork_management > fork > signalling_block_count' greater than signalling interval",
+       fun invalid_fork_signalling_block_count/0},
+      {"User configuration cannot contain 'fork_management > fork > fork_height' lower or equal to 'fork_management > fork > signalling_end_height'",
+       fun invalid_fork_height/0},
+      {"User configuration cannot contain 'fork_management > fork > version' lower or equal to Lima protocol version (4)",
```

Comment:
I [would rather expect Minerva](https://github.com/aeternity/aeternity/pull/2613/files#diff-119c822db20ee796b91225c284221d8eR15).

Reply:
Because of info field.

Reply:
Strictly greater than Minerva.

Reply:
Fixed.